### PR TITLE
Multisite

### PR DESCRIPTION
**Pull request by gignonje**

### CODE REVIEW

File: adback-solution-to-adblock/class-ad-back.php [hunk 1]
```diff
@@ -162,7 +162,7 @@ public function getDomain()
         global $wpdb; // this is how you get access to the database
 
         $table_name = $wpdb->prefix . 'adback_myinfo';
```
Comment (nicolasThal) [inline adback-solution-to-adblock/class-ad-back.php]: I think you should add the blog id in the `table_name`

**Comment by sempixel** [comment]
Are you sure about individual and network wide activation?

**Comment by gignonje** [comment]
@semp I'll check both individual and blog creation

File: adback-solution-to-adblock/includes/class-ad-back-activator.php [hunk 2]
```diff
@@ -91,43 +115,30 @@ public static function activate()
...
-        ) ".$charset_collate.";";
-
-        dbDelta( $sql );
-
         $wpdb->insert(
-            $table_name,
+            $table_name_info,
             array(
-                "id" => "1",
+                "id" => $blogId,
```
Comment (nicolasThal) [inline adback-solution-to-adblock/includes/class-ad-back-activator.php]: as you are creating one table per blog, I am not sure that you still needs to set the id to the `$blogId`

Comment (gignonje) [inline adback-solution-to-adblock/includes/class-ad-back-activator.php]: I think it to be more meaningful that id = 1 for every table

**Pull request merged**